protos/update_protos.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# NOTE:: It would be better if we just pull in the helium proto repo as a submodule.
```

Comment:
removed.

Reply:
our protos are out of sync because nothing keeps them updated. A submodule will show the with simple git commands. If I download the latest blockchain_txn.proto all these files will be missing: 
```
import "blockchain_txn_coinbase_v1.proto";
import "blockchain_txn_security_coinbase_v1.proto";
import "blockchain_txn_oui_v1.proto";
import "blockchain_txn_gen_gateway_v1.proto";
import "blockchain_txn_routing_v1.proto";
import "blockchain_txn_payment_v1.proto";
import "blockchain_txn_security_exchange_v1.proto";
import "blockchain_txn_consensus_group_v1.proto";
import "blockchain_txn_consensus_group_failure_v1.proto";
import "blockchain_txn_add_gateway_v1.proto";
import "blockchain_txn_assert_location_v1.proto";
import "blockchain_txn_create_htlc_v1.proto";
import "blockchain_txn_redeem_htlc_v1.proto";
import "blockchain_txn_poc_request_v1.proto";
import "blockchain_txn_poc_receipts_v1.proto";
import "blockchain_txn_vars_v1.proto";
import "blockchain_txn_rewards_v1.proto";
import "blockchain_txn_token_burn_v1.proto";
import "blockchain_txn_dc_coinbase_v1.proto";
import "blockchain_txn_token_burn_exchange_rate_v1.proto";
import "blockchain_txn_state_channel_open_v1.proto";
import "blockchain_txn_update_gateway_oui_v1.proto";
import "blockchain_txn_state_channel_close_v1.proto";
import "blockchain_txn_payment_v2.proto";
import "blockchain_txn_price_oracle_v1.proto";
import "blockchain_txn_gen_price_oracle_v1.proto";
import "blockchain_txn_transfer_hotspot_v1.proto";
import "blockchain_txn_rewards_v2.proto";

import "blockchain_txn_gen_validator_v1.proto";
import "blockchain_txn_stake_validator_v1.proto";
import "blockchain_txn_transfer_validator_stake_v1.proto";
import "blockchain_txn_unstake_validator_v1.proto";
import "blockchain_txn_validator_heartbeat_v1.proto";

import "blockchain_txn_assert_location_v2.proto";
import "blockchain_txn_transfer_hotspot_v2.proto";
```

The other options is to evaluated whether we even need this proto.

Reply:
resolved during daily standup. Manual copying continues.